### PR TITLE
fix: revert to single-value video gallery dropdown filter

### PR DESCRIPTION
**Pull request by CefBoud**
# Description

This PR affects the `SelectionModal` component used by `VideoGallery` and `SelectImageModal` components.

It reverts changes added by #368.  The goal is to have a uniform filter & sort UI.

# Before
image

# After
image

# Testing instructions

1. (Optional) Create a mock videos response with some broken thumbnails
2. Enable mock video uploads (create [waffle flag](http://localhost:18010/admin/waffle/flag/) contentstore.mock_video_uploads enabled for everyone)
3. Modify [videos_index_json()](https://github.com/openedx/edx-platform/blob/565b34e4e0904d1a55e56db403893a207c62e4c3/cms/djangoapps/contentstore/video_storage_handlers.py#L692) as follows:
```python
def videos_index_json(course):
    index_videos = [
        {
            'edx_video_id': 'aaaaaaaa-aaaa-4aaa-aaaa-aaaaaaaaaaaa',
            'client_video_id': 'video.mp4',
            'created': '1970-01-01T00:00:00Z',
            'duration': 42.5,
            'status': 'Ready',
            'course_video_image_url': 'https://video/images/1234.jpg'
        },
        {
            'edx_video_id': 'aaaaaaaa-aaaa-4aaa-aaaa-aaaaaaaaaaaa',
            'client_video_id': 'video.mp4',
            'created': '1970-01-01T00:00:00Z',
            'duration': 42.5,
            'status': 'Failed',
            'course_video_image_url': 'https://video/images/1234.jpg'
        }
    ]
    return JsonResponse({"videos": index_videos}, status=200)
```
4.  Enable the new video editor (create [waffle flags](http://localhost:18010/admin/waffle/flag/) `new_core_editors.use_new_video_editor` and `new_core_editors.use_video_gallery_flow` for everyone)
Create a course unit with a video
5. Open the video gallery  http://localhost:2001/course/course-v1:edX+DemoX+Demo_Course/editor/course-videos/videoBlockId

6.  Filter by status using the single-value filter dropdown.

private ref: BB-8054

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @CefBoud!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: src/editors/sharedComponents/SelectionModal/MultiSelectFilterDropdown.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const MultiSelectFilterDropdown = ({
 }) => {
   const intl = useIntl();
   return (
-    <Dropdown autoClose={false}>
+    <Dropdown autoClose="outside">
```
Comment (farhaanbukhsh) [inline src/editors/sharedComponents/SelectionModal/MultiSelectFilterDropdown.jsx:14]: Can we convert this into a prop and pass it from the component?

**Comment by farhaanbukhsh** [comment]
@CefBoud  please update the description with the template in the handbook. You can have a look at https://github.com/openedx/frontend-lib-content-components/pull/412.

**Review by farhaanbukhsh — APPROVED** [review]
👍 

- ✅ I tested this on the devstack.
- ✅  I read through the code
- ❌  I checked for accessibility issues
- ❌  Includes documentation
- ❌  I made sure any change in configuration variables is reflected in the corresponding client's `configuration-secure` repository.

**Comment by farhaanbukhsh** [comment]
@CefBoud can you remove the draft status please.

**Comment by CefBoud** [comment]
@farhaanbukhsh Description fixed and status changed.

**Comment by tecoholic** [comment]
@navinkarkera Another request to add the "jira:2u" label. :)

cc: @cablaa77

**Comment by mphilbrick211** [comment]
Hi @openedx/teaching-and-learning! This is ready for review. Thanks!

**Comment by openedx-webhooks** [comment]
I've created issue [TNL-11222](https://2u-internal.atlassian.net/browse/TNL-11222) in the private 2U Jira.

**Comment by tecoholic** [comment]
@e0d Yes. This is related to the Video Block UX improvements project work.

**Review by KristinAoki — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@CefBoud 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-lib-content-components&entry.867055334=https://github.com/openedx/frontend-lib-content-components/pull/414&entry.1484655318=https://github.com/CefBoud&entry.752974735=2023-10-23+08:48&entry.1917517419=2023-11-30+19:43&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**